### ISSUE TITLE
UnusedPrivateMember: Incorrectly reports unused params in open/abstract functions

### ISSUE THREAD

**Issue by marianosimone** [arturbosch/detekt#972]
The rule is over-reporting

For the following example:

```
open class Foo {
    abstract fun foo(bar: Bar): Unit
}

class Foo2 : Foo {
    override fun foo(bar: Bar) {
        bar.bar()
    }
}
```

## Expected Behavior

Nothing should be reported

## Current Behavior

reports `bar` in `Foo#foo` as unused

This seems related to (my) changes in #968, and somehow got released to RC7-2 (instead of to the recently release RC7-3)

**Comment by marianosimone** [comment]
I'm working on a fix for this right now

**Comment by arturbosch** [comment]
Thanks for the fast fix!
